Makefile.include

```diff
@@ -290,6 +290,13 @@ term: $(filter flash, $(MAKECMDGOALS))
 doc:
 	make -BC $(RIOTBASE) doc
 
+btest:
```

Comment:
btest is a "board unit test" that runs a python script executed by gdb python engine.

make BOARD=<board> TEST_SUITE=mysuite

will execute the python file $(CURDIR)/tests/mysuite.py

where mysuite.py is something like:

	import unittest

	int_pointer_type=gdb.lookup_type('int').pointer()

	TIMER3_TAR=0x40033048


	def mem_content(address):
		addr_val = gdb.Value(address)
		addr_ptr = addr_val.cast(int_pointer_type)
		return long(addr_ptr.dereference())

	class HwTimerTest(unittest.TestCase):

		def test_hwtimer_wait(self):
			gdb.execute("break main.c:55")
			gdb.execute("continue")
			start_time=mem_content(TIMER3_TAR)
			gdb.execute("break +1")
			gdb.execute("continue")
			end_time=mem_content(TIMER3_TAR)
			print(end_time-start_time)

			self.assertLessEqual(4000000, end_time-start_time)
			self.assertLessEqual(end_time-start_time, 4002000)

			# get SP register value
			#print(gdb.parse_and_eval("$sp"))
			#print(gdb.execute("x 0x40033048"))

	unittest.main()


This is just a beginning tentative of using python with gdb.